docs/guides/using_filters.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: home
+title: Using filters to narrow your search
+parent: How-to guides
+nav_order: 5
+---
+
+## Using filters to narrow your search
+
+TIMDEX includes a set of filters that you can use to limit your search. The [API reference](https://mitlibraries.github.io/timdex/reference/#query-search)
+for the search operation lists each available filter and how to use them. In this how-guide, we'll look at the
+sourceFilter, but the process is similar with other filters. If you're not sure what values a filter will accept, try
+a search that [returns aggregations](viewing_aggregations).
+
+This example demonstrates a query that searches for the term "data" in the two sources dspace@mit and zenodo. We are
```

Comment:
Let's use "not Zenodo" in this example. dspace and alma or dspace and aspace as I don't recall if we'll leave zenodo in as a default search (I think it's supposed to require a separate index but we may not have configured our endpoints to do that yet...)